CHANGELOG.md

```diff
@@ -37,6 +37,8 @@ Bug Fixes
   images.
 - Updated ups table file so that setup command is `setup galsim` instead of
   `setup GalSim` (#724)
+- Improved utilities.thin_tabulated_values() (used by SED.thin and
```

Comment:
Just my two cents: I recommend making this more descriptive and less specific about the routine (i.e., "Improved algorithm for thinning SEDs and bandpasses to enable faster calculations while still meeting relative error constraints") or something like that.